services/gateway/apis/http-version3/swagger/parameters/transactions.json

```diff
@@ -15,6 +15,15 @@
             "$ref": "#/definitions/transactionParamEnvelope"
         }
     },
+    "dryrunTransaction": {
+        "name": "body",
```

Comment:
```suggestion
        "name": "transaction",
```

Reply:
I think, it already is correct the way it is. Please re-verify.
@priojeetpriyom Please remove `accountInitializationFee` from the transactions params to avoid future rework. In the subsequent release this will be removed by the SDK as all the extra fees will now be part of the transaction fee.

Reply:
changed to `dryrunTransaction` instead